server.go

```diff
@@ -243,6 +245,33 @@ func (s *Server) register(sd *ServiceDesc, ss interface{}) {
 	s.m[sd.ServiceName] = srv
 }
 
+// ServiceInfo contains method names and metadata for a service.
+type ServiceInfo struct {
+	Methods  []string
```

Comment:
comment on whether this is the full method name (package + service + method) or method only.